Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM node:carbon
+
+RUN apt-get update \
+    && apt-get install -y apt-utils \
+    && { \
+        echo debconf debconf/frontend select Noninteractive; \
+        echo mysql-community-server mysql-community-server/data-dir \
+            select ''; \
+        echo mysql-community-server mysql-community-server/root-pass \
+            password '123'; \
+        echo mysql-community-server mysql-community-server/re-root-pass \
+            password '123'; \
+        echo mysql-community-server mysql-community-server/remove-test-db \
+            select true; \
+    } | debconf-set-selections \
+    && apt-get install -y mysql-server
+
+# Ensure we won't bind to localhost only
+RUN grep -v bind-address /etc/mysql/my.cnf > temp.txt \
+  && mv temp.txt /etc/mysql/my.cnf
+
+# It doesn't seem needed since I'll use -p, but it can't hurt
+EXPOSE 3306
+
+CMD /etc/init.d/mysql start && tail -F /var/log/mysql.log
+
+# Create app directory
+WORKDIR /home/ayush/WebstormProjects/Atlas/openmrs-contrib-atlas/
```

Comment:
No. This is incorrect. Paths within docker hang off of `/` unless you specify otherwise. 

Make this `/opt/atlas` or something.